Parent/Parent/ParentAppDelegate.swift

```diff
@@ -28,7 +28,11 @@ import SafariServices
 
 let ParentAppRefresherTTL: TimeInterval = 5.minutes
 var legacySession: Session?
-var currentStudentID: String?
+var currentStudentID: String? {
+    didSet {
+        ParentAppDelegate.persistCurrentStudent(id: currentStudentID)
```

Comment:
I think it makes more sense to use [SessionDefaults](https://github.com/instructure/canvas-ios/blob/master/Core/Core/AppEnvironment/SessionDefaults.swift) for this. That's where we store user preferences like this.